deploy/kubernetes/kube-prometheus-sumo-logic-mixin.libsonnet

```diff
@@ -6,7 +6,7 @@
   sumologicCollector:: {
     remoteWriteConfigs+: [
       {
-        url: $._config.sumologicCollectorSvc + "prometheus.metrics.state",
+        url: "http://$(CHART).$(NAMESPACE).svc.cluster.local:9888/prometheus.metrics.state",
```

Comment:
We'll need to fix this generated file in the CI

Reply:
Done